src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -1544,22 +1538,23 @@ class TC_GAME_API SmartWaypointMgr
 {
     private:
         SmartWaypointMgr() { }
-        ~SmartWaypointMgr();
+        ~SmartWaypointMgr() { }
 
     public:
         static SmartWaypointMgr* instance();
 
         void LoadFromDB();
 
-        WPPath* GetPath(uint32 id)
+        SmartWaypointPath const* GetPath(uint32 id)
         {
-            if (waypoint_map.find(id) != waypoint_map.end())
-                return waypoint_map[id];
-            else return nullptr;
+            auto itr = waypointMap.find(id);
+            if (itr != waypointMap.end())
+                return &itr->second;
+            return nullptr;
         }
 
     private:
-        std::unordered_map<uint32, WPPath*> waypoint_map;
+        std::unordered_map<uint32, SmartWaypointPath> waypointMap;
```

Comment:
I see you renamed this field but thats still not how class members are supposed to be named - https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2130103